src/org/sage_one_sample/sageone/SageoneToken.java

```diff
@@ -6,77 +6,142 @@
+			resp.getWriter().println("<textarea id='data' class='form-control' name='data'></textarea>");
+			resp.getWriter().println("<p>Example: {'contact[contact_type_id]': 1, 'contact[name]': 'Joe Bloggs'}</p>");
+			resp.getWriter().println("<input name='access_token' type='hidden' value='" + token + "'>");
+			resp.getWriter().println("<input type='submit' value='POST' class='btn btn-primary'>");
+			resp.getWriter().println("</form>");
+			resp.getWriter().println("</div>");
+			resp.getWriter().println("<div role='tabpanel' class='tab-pane fade' id='put'>");
+			resp.getWriter().println("<h3>PUT request</h3>");
+			resp.getWriter().println("<form action='/SageOneSampleApp/sageone_data' method='post'>");
+			resp.getWriter().println("<input name='request_method' type='hidden' value='put'>");
+			resp.getWriter().println("<label for='endpoint'>Endpoint</label>");
+			resp.getWriter().println("<input name='endpoint' type='text' class='form-control' required='true'>");
+			resp.getWriter().println("<p>Example: accounts/v1/contacts/:id</p>");
+			resp.getWriter().println("<label for='data'>Post data</label>");
+			resp.getWriter().println("<textarea id='data' class='form-control' name='data'></textarea>");
+			resp.getWriter().println("<p>Example: {'contact[contact_type_id]': 1, 'contact[name]': 'Joe Bloggs'}</p>");
+			resp.getWriter().println("<input name='access_token' type='hidden' value='" + token + "'>");
+			resp.getWriter().println("<input type='submit' value='PUT' class='btn btn-primary'>");
+			resp.getWriter().println("</form>");
+			resp.getWriter().println("</div>");
+			resp.getWriter().println("<div role='tabpanel' class='tab-pane fade' id='delete'>");
+			resp.getWriter().println("<h3>DELETE request</h3>");
+			resp.getWriter().println("<form action='/SageOneSampleApp/sageone_data' method='get'>");
+			resp.getWriter().println("<input name='request_method' type='hidden' value='delete'>");
+			resp.getWriter().println("Endpoint: <input name='endpoint' type='text' class='form-control' required='true'>");
+			resp.getWriter().println("<p>Example: accounts/v1/contacts/:id</p>");
+			resp.getWriter().println("<input name='data' type='hidden' value=''>");
+			resp.getWriter().println("<input name='access_token' type='hidden' value='" + token + "'>");
+			resp.getWriter().println("<input type='submit' value='DELETE' class='btn btn-primary'>");
+			resp.getWriter().println("</form>");
+			resp.getWriter().println("</div>");
+			resp.getWriter().println("</div>");
+			resp.getWriter().println("</div>");
+			resp.getWriter().println("</div>");
+			resp.getWriter().println("</body>");
+			resp.getWriter().println("</html>");
+		} catch (IOException ex) {
+			ex.printStackTrace();
+		}
+	}
+
+	/**
+	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
this is how a servlet is generated by default - 2 methods `doGet` and `doPost` with `doPost` simply calling `doGet`. Not entirely sure why though.